src/netspresso_trainer/dataloaders/base.py

```diff
@@ -95,6 +95,9 @@ def _load_dataset(self, root, subset_name=None, cache_dir=None):
         total_dataset = load_dataset(root, name=subset_name, cache_dir=cache_dir)
         return total_dataset
 
+    def cache_dataset(self):
+        pass
+
```

Comment:
Not defined base class method should be set as `@abstractmethod`

```suggestion
    @abstractmethod
    def cache_dataset(self):
        pass

```

Reply:
And, please set extended class's `cache_data` as `NotImplemented`.
(or just log warning)